features/environment.py

```diff
@@ -170,6 +170,11 @@ def _make_patroni_test_config(self, name, custom_config):
             config = yaml.safe_load(f)
             config.pop('etcd', None)
 
+        raft_port = os.environ.get('RAFT_PORT')
+        if raft_port:
+            os.environ['RAFT_PORT'] = str(int(raft_port) + 1)
```

Comment:
Yeah curious about this one.

Reply:
This file is responsible for creating environment for integration tests.
Since during integration testing all Patroni are running on on the same host we have to make sure that we always listening on different ports.
We are are also doing pretty much the same with postgres port: https://github.com/zalando/patroni/pull/375/files/dbdf55acfba50f43615e2bb7fa7f407df67e9e72#diff-c27f616ff69b00881ddec0da23e0cd89R94